getting_started_tutorials/cudf_pandas_stocks_demo.ipynb

```diff
@@ -0,0 +1,7061 @@
+{
```

Comment:
<p>@beckernick, do we want to say anything special about the chart?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>